src/sources.js

```diff
@@ -294,13 +294,18 @@ class S3Source extends Source {
     return `https://${this.bucket}.s3.amazonaws.com`;
   }
   async _listObjects() {
-    // XXX TODO: This will only return the first 1000 objects.  That's fine for
-    // now (for comparison, nextstrain-data only has ~500), but we really
-    // should iterate over the whole bucket contents using the S3 client's
-    // pagination support.
-    //   -trs, 30 Aug 2019
-    const list = await S3.listObjectsV2({Bucket: this.bucket}).promise();
-    return list.Contents;
+    const s3Params = {Bucket: this.bucket, MaxKeys: 1000};
```

Comment:
The [AWS SDK](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#listObjectsV2-property) documents that the default `MaxKeys` is 1000, so this change is not necessary.

Reply:
I have removed this on your suggestion, however I originally chose to include it as I preferred the explicitness this provided at the small cost of verbosity.

Reply:
Fair enough. This is why I called it a nitpick. I accept a difference of opinion here, and I would not make something like that a blocker on this PR :slightly_smiling_face: